sunpy/map/tests/test_map_factory.py

```diff
@@ -154,7 +156,13 @@ def test_sot(self):
         assert isinstance(sot,sunpy.map.sources.SOTMap)
 
         #Test SWAPMap
+        swap = sunpy.map.Map(os.path.join(filepath, "SWAP", "resampled0_swap.fits"))
+        assert isinstance(swap, sunpy.map.sources.SWAPMap)
 
         #Test XRTMap
+        xrt = sunpy.map.Map(os.path.join(filepath, "HinodeXRT.fits"))
+        assert isinstance(xrt, sunpy.map.sources.XRTMap)
```

Comment:
I would keep this test here if `test_sot` is renamed to `test_hinode` and includes `sot` and `xrt`